android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -69,6 +69,11 @@ private Boolean isTablet() {
     return layout == Configuration.SCREENLAYOUT_SIZE_LARGE || layout == Configuration.SCREENLAYOUT_SIZE_XLARGE;
   }
 
+  @ReactMethod
+  public void getDeviceInfo(@NonNull final Promise promise) {
+    promise.resolve(this.getConstants().get("DeviceInfo"));
+  }
+
   @Override
   public @Nullable Map<String, Object> getConstants() {
```

Comment:
Perhaps if we had built this from scratch we might have gone that route, but there is a lot of information in this module that is not going to change, aside from what we've recently added; I think that's why they had set it up that way.

We can open an issue to discuss the future of this fork, however this PR is not looking to make drastic changes to what the origin of this fork has offered, just makes it accessible and compatible for our purposes.